content/english/security/4-spy-game/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Spy Game"
+description: ""
+draft: false
+weight: 4
+translationKey: "Security-4"
+---
+
+
+From our last activity we learnt that a strong password is a combination of random characters (possibly having uppercase and lowercase letters), numbers and some special characters. Now it's time to play Hacked or Safe!
+
+From a list of passwords used by our `Nuvi` select the strongest password which cannot be hacked by **Eve**.
+ 
+
+![Cannot load image](pwd1.JPG?classes=border,shadow)
+<fieldset>  
+  <legend>Can this password be Hacked?</legend>
+  <input type="radio" id="notify-on" name="notify" value="on" checked>
+  <label for="notify-on">Yes</label>
+  <input type="radio" id="notify-off" name="notify" value="off">
+  <label for="notify-off">No</label>
+</fieldset> 
+
+
+
+![Cannot load image](pwd2.JPG?classes=border,shadow)
+
+
+<fieldset>  
+  <legend>Can this password be Hacked?</legend>
+  <input type="radio" id="notify-on" name="notify" value="on" checked>
+  <label for="notify-on">Yes</label>
+  <input type="radio" id="notify-off" name="notify" value="off">
+  <label for="notify-off">No</label>
+</fieldset>
+
+![Cannot load image](pwd3.JPG?classes=border,shadow)
+
+<fieldset>  
+  <legend>Can this password be Hacked?</legend>
+  <input type="radio" id="notify-on" name="notify" value="on" checked>
+  <label for="notify-on">Yes</label>
+  <input type="radio" id="notify-off" name="notify" value="off">
+  <label for="notify-off">No</label>
+</fieldset> 
+
+![Cannot load image](pwd4.JPG?classes=border,shadow)
+
+<fieldset>  
+  <legend>Can this password be Hacked?</legend>
+  <input type="radio" id="notify-on" name="notify" value="on" checked>
+  <label for="notify-on">Yes</label>
+  <input type="radio" id="notify-off" name="notify" value="off">
+  <label for="notify-off">No</label>
+</fieldset> 
+
+<!--
```

Comment:
Same thing here - can we just delete this adafruit related work?

Reply:
yup!